### ISSUE TITLE
huge npm package > 7Mb

### ISSUE THREAD

**Issue by drortirosh** [jimhigson/oboe.js#228]
for "dependency-less, lightweight" library, 7Mb package is a bit too large.
I suggest not packing entire test suite into the npm package

**Comment by ScottAwesome** [comment]
not really sure what you mean? according to bundlephobia, its 17KB minified, 5.5KB gzipped: https://bundlephobia.com/result?p=oboe@2.1.5

if you mean don't ship unnecessary directories, I can understand that, but this is not clear in your issue title.

**Comment by AcidRaZor** [comment]
check the size of your node_modules directory. its known to be huge

**Comment by drortirosh** [comment]
yes, the node_modules is huge - for this exact reason (many small packages that don't care about their impact on the total node_modules size...)
